apps/experiments/models.py

```diff
@@ -288,6 +288,20 @@ class VoiceResponseBehaviours(models.TextChoices):
     NEVER = "never", gettext("Never")
 
 
+class AgentTools(models.TextChoices):
+    RECURRING_REMINDER = "recurring-reminder", gettext("Recurring Reminder")
+    ONE_OFF_REMINDER = "one-off-reminder", gettext("One-off Reminder")
+    SCHEDULE_UPDATE = "schedule-update", gettext("Schedule Update")
+
+
+class AgentToolResource(BaseModel):
```

Comment:
Is it necessary to have a new model? You could make the field on the experiment an ArrayField (e.g. https://github.com/dimagi/open-chat-studio/blob/f17abadda0a2b957065069aa674efc8cf2782e30/apps/assistants/models.py#L33)

Reply:
https://github.com/dimagi/open-chat-studio/pull/417/commits/ab100e7fdfd262484b99080495928bfb2e5ad4c1